### ISSUE TITLE
ability to export svgs as json

### ISSUE THREAD

**Issue by chrisdrackett** [souporserious/unicon#11]
right now the json exporter exports a `.js` file. It would be useful if doing further work on these svgs to be able to work on a `.json` file instead.

**Comment by souporserious** [comment]
Good call! I need to explore this more. I created [this issue](https://github.com/souporserious/unicon/issues/3) to discuss transformers in more depth. I'm not sure the best way to structure adding options for individual transforms. Maybe `unicon-transformer-json` should handle this use case and I can create another package in place called `unicon-transformer-json-exports` or something.

**Comment by chrisdrackett** [comment]
yeah, I was thinking that `unicon-transformer-json` is a pretty good name for what I'm suggesting. I was going to start working on this but saw some of the file generation isn't within the transformer.

I need to think about it some, but I think it might make sense for the transformer to handle the file generation as well.

Should generating react components be handled by a transformer for example?

**Comment by souporserious** [comment]
Yup! A transformer definitely could handle that. I'm taking a different approach by exporting the JSON fragments, but I think a `unicon-transformer-react` package might make sense.
